docs/guides/magicgui.md

```diff
@@ -0,0 +1,530 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.8.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+(magicgui)=
```

Comment:
this line shows up when I preview the markdown file in VSCode - is that supposed to happen? Is it just a vscode quirk?

Reply:
yeah, the VSCode preview engine probably isn't running the whole myst-md parser that will be used when transpiling this document in jupyter-books, so a lot of it will look different.